src/PostgREST/Middleware.hs

```diff
@@ -100,7 +104,9 @@ pgrstFormat minStatus date req status responseSize =
   if status < minStatus
     then mempty
   else toLogStr (getSourceFromSocket req)
-    <> " - - ["
+    <> " "
+    <> toLogStr (maybe "-" unquoted (getRoleClaim req))
+    <> " - ["
```

Comment:
Since I'd like to remove `pgrstFormat` in #1724, I opened a PR for `wai-logger` to add upstream support for the current user in the apache logger. See https://github.com/kazu-yamamoto/logger/pull/198 for details.

Reply:
@wolfgangwalther  Just saw https://github.com/kazu-yamamoto/logger/pull/198 was merged :raised_hands:. Nice work!